src/ScheduledTask/ProductImportTask.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace ShopwareBlog\FastBillConnector\ScheduledTask;
+
+use Shopware\Core\Framework\MessageQueue\ScheduledTask\ScheduledTask;
+
+class ProductImportTask extends ScheduledTask
+{
+    public static function getTaskName(): string
+    {
+        return 'shopware-blog.import-products';
+    }
+
+    public static function getDefaultInterval(): int
+    {
+        return 5;
```

Comment:
Wenn mich nicht alles täuscht ist das in sekunden :D willst du den import alle 5 sekunden anstoßen?

Reply:
Si zum testen ja :-D